packages/components/src/radio-control/README.md

```diff
@@ -1,11 +1,59 @@
 # RadioControl
 
-RadioControl component is used to generate radio input fields.
+Use radio buttons when you want users to select one option from a set, and you want to show them all the available options at once.
 
+![Selected and unselected radio buttons](./radio.png)
+Selected and unselected radio buttons
+
+## Table of contents
+
+1. [Design guidelines](#design-guidelines)
+2. [Development guidelines](#development-guidelines)
+3. [Related components](#related-components)
+
+# Design guidelines
+
+## Usage
+
+### When to use radio buttons
+
+Use radio buttons when you want users to:
+
+- Select a single option from a list.
+- Expose all available options.
+
+If you have a list of available options can be collapsed, consider using a dropdown menu instead — it uses less space.
+
+![Use radio buttons when only one item can be selected from a list.](./radio-usage-do.png)
```

Comment:
I'm not sure the `alt` attribute text should duplicate the text label. My understanding is that `alt` should describe what users would "see" in the image, but this text sort of augments the image without clearly explaining what the image depicts.